Makefile

```diff
@@ -1,2 +1,56 @@
+all: whatdoesitdo tests authors
+
 whatdoesitdo:
+	@# generate the README "What does it do?" section
 	@./gen-readme.sh
+
+# To decrease tests verbosity, comment out unneeded targets
+tests: downloadffprefs checknotcovered checkdeprecated stats cleanup
```

Comment:
I forgot to post the link I had found: http://stackoverflow.com/questions/4346399/how-can-i-force-gnu-make-to-not-build-recipe-in-parallel/23903293#23903293

```
.PHONY: all clean

.NOTPARALLEL:

anotherTarget: dependency1
```

Apparently the MAKEFLAGS variable can also be used (solution 3)